optimum/exporters/onnx/model_configs.py

```diff
@@ -256,6 +257,23 @@ class ImageGPTOnnxConfig(GPT2OnnxConfig):
     pass
 
 
+class DecisionTransformerOnnxConfig(GPT2OnnxConfig):
+    DUMMY_INPUT_GENERATOR_CLASSES = (
+        DummyDecisionTransformerInputGenerator,
+    )
+
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        dynamic_axis = {0: "batch_size", 1: "sequence_length"}
+
+        return {
+            'actions': dynamic_axis,
+            'timesteps': dynamic_axis,
+            'attention_mask': dynamic_axis,
+            'returns_to_go': dynamic_axis,
+            'states': dynamic_axis,
+        }
```

Comment:
I think this is wrong, can you investigate what are the actual dimensions of these inputs

Reply:
Input dimensions are mentioned in input_generator.py. 

```
    def generate(self, input_name: str, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):

        if input_name == "states":
            shape = [self.batch_size, self.normalized_config.config.state_dim]
        elif input_name == "actions":
            shape = [self.batch_size, self.normalized_config.config.act_dim]
        elif input_name == 'returns_to_go':
            shape = [self.batch_size, 1]
        elif input_name == 'timesteps':
            shape = [self.normalized_config.config.state_dim, self.batch_size]
            max_value = self.normalized_config.config.max_ep_len
            return self.random_int_tensor(shape=shape, max_value = max_value, framework=framework, dtype=int_dtype)
        elif input_name == "attention_mask":
            shape = [self.batch_size, self.normalized_config.config.state_dim]

        return self.random_float_tensor(shape, min_value=-1., max_value=1., framework=framework, dtype=float_dtype)
```

When I debug, input shapes as mentioned above are correctly cascaded to the decision transformer model.

Reply:
yes but the dynamic axis are wrong, they should be specified correctly in the generator and in the model config.

Reply:
Have made changes for actual dimensions, please review.